src/Commands/Tools/ToolsListCommand.cs

```diff
@@ -49,14 +48,15 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
 
     private static CommandInfo CreateCommand(string hyphenatedName, IBaseCommand command)
```

Comment:
We have a separate bug report that states hyphenated command names are invalid and we should be snake_casing. Do you want to fix that at the same time?